.github/workflows/mac.yml

```diff
@@ -17,11 +17,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 11
+    - run: curl -OL "https://corretto.aws/downloads/latest/amazon-corretto-11-x64-macos-jdk.tar.gz"
```

Comment:
Can you add a comment in the file on why this is here so we can remove it when they upgrade the jvm version?

Reply:
why remove it? I figure we could continue using corretto...

Reply:
Speed? But, fair it should be fine either way

Reply:
If not a source comment,  mentioning the actual bug in the commit message is at least worthwhile.